server/handlers.go

```diff
@@ -1034,6 +1034,11 @@ func (s *Server) getHandler(w http.ResponseWriter, r *http.Request) {
 
 	if action == "inline" {
 		disposition = "inline"
+		/*
```

Comment:
the comment does not seem correct :)

also, I would expect https://github.com/dutchcoders/transfer.sh/pull/501/files#diff-961509ed3ff668f3172148150903514fb3394d6ff090d6db78d986875b13689fR1056-R1058 to sanitise the content and avoid XSS

if the above is not working it should be fixed there (or on the bluemonday module)